ext-apiserver/pkg/controller/machineset/controller_test.go

```diff
@@ -33,13 +33,25 @@ func machineSetControllerReconcile(t *testing.T, cs *clientset.Clientset, contro
 	instance.Name = "instance-1"
 	replicas := int32(0)
 	instance.Spec.Replicas = &replicas
-	instance.Spec.Selector = metav1.LabelSelector{MatchLabels: map[string]string{"foo":"barr"}}
-	instance.Spec.Template.Labels = map[string]string{"foo":"barr"}
+	instance.Spec.Selector = metav1.LabelSelector{MatchLabels: map[string]string{"foo": "barr"}}
+	instance.Spec.Template.Labels = map[string]string{"foo": "barr"}
 
 	expectedKey := "default/instance-1"
 
 	// When creating a new object, it should invoke the reconcile method.
-	cluster := v1alpha1.Cluster{}
+	cluster := v1alpha1.Cluster{
```

Comment:
Instead of re-declaring this vanilla cluster every test, perhaps it would make sense to have a method return the vanilla Cluster?

Reply:
That or have some const?

Reply:
I think it's a fine suggestion. I'm having a hard time to find a place to define this const though. Will syncup offline.